packages/core/src/components/Editable/index.js

```diff
@@ -47,6 +48,7 @@ type PropTypes = {
 class Editable extends Component {
   constructor(props: PropTypes) {
     super(props)
+    this.previousState = !props.id ? null : editable(props.editor.store.getState(), { id: props.id })
```

Comment:
I think this can lead to a race condition where the component is being mounted but the editor state is not yet available. That's the reason why previousState was initialized with null

Reply:
Not sure if i get it 😅, how this can be happened ?

Reply:
Only because the editable is being rendered does not mean that the store has the editable already - it might be loaded asynchronously from a backend or the js engine might decide to execute this code before updating the internal store.